storage/src/main/scala/io/casperlabs/storage/deploy/SQLiteDeployStorage.scala

```diff
@@ -321,6 +323,22 @@ class SQLiteDeployStorage[F[_]: Metrics: Time: Sync](chunkSize: Int)(
     } yield res
   }
 
+  override def getDeploysByAccount(
+      account: ByteString,
+      limit: Int,
+      lastTimeStamp: Long,
+      lastDeployHash: DeployHash
+  ): F[List[Deploy]] =
+    sql"""|SELECT data FROM deploys
+					|WHERE account = $account
```

Comment:
```suggestion
          |WHERE account = $account
```
You should set your editor to use spaces for tabs.

Reply:
Done.